applications/FluidDynamicsApplication/trilinos_extension/TrilinosExtension.py

```diff
@@ -0,0 +1,4 @@
+from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+import KratosMultiphysics.TrilinosApplication
```

Comment:
is this necessary here?
Guess so bcs otherwise you potentially have undefined symbols right?

Reply:
yes, that's the case here. If I remember correctly, it is because some classes (schemes in particular) derive from base classes exposed from TrilinosApplication.